src/main/java/org/benetech/mathshare/Application.java

```diff
@@ -10,7 +10,8 @@
 
 @SpringBootApplication
 //TODO: Remove EnableAutoConfiguration excluded classes when there will be data source attached
-@EnableAutoConfiguration(exclude={DataSourceAutoConfiguration.class,HibernateJpaAutoConfiguration.class})
+@EnableAutoConfiguration(exclude = {DataSourceAutoConfiguration.class, HibernateJpaAutoConfiguration.class})
+@SuppressWarnings("checkstyle:hideutilityclassconstructor")
```

Comment:
can't we just fix the warning?

Reply:
Unfortunately, the Checkstyle treats our spring main class as a utility class - it has only public, static methods. Those classes should not expose public constructor, as they shouldn't be instantiated. In our case, we need this public constructor.